lib/partition_setup.pm

```diff
@@ -404,28 +404,49 @@ sub take_first_disk_storage_ng {
     select_first_hard_disk;
 
     assert_screen [qw(existing-partitions partition-scheme)];
-    # If drive is not formatted, we have select hard disks page
-    # On ipmi we always have unformatted drive
-    # Sometimes can have existing installation on iscsi
+    # If drive(s) is/are not formatted, we have select hard disks page
     if (match_has_tag 'existing-partitions') {
-        send_key $cmd{next};
-        assert_screen 'partition-scheme';
+        if (check_var('BACKEND', 'ipmi')) {
+            send_key_until_needlematch("remove-menu", "tab");
+            while (check_screen('remove-menu', 3)) {
+                send_key 'spc';
+                send_key 'down';
+                send_key 'ret';
+                send_key 'tab';
+            }
+            save_screenshot;
+            send_key $cmd{next};
+            if (check_screen([qw(filesystems-options partition-scheme)], 3)) {
+                send_key $cmd{next};
+            }
+            if (check_screen([qw(filesystems-options partition-scheme)], 3)) {
+                send_key $cmd{next};
+            }
+            assert_screen "after-partitioning";
+        }
+        else {
+            send_key $cmd{next};
+            assert_screen 'partition-scheme';
+        }
```

Comment:
@mloviska @jknphy @OleksandrOrlov @rwx788 feedback regarding the above section or the changes within this file?

Reply:
@mloviska @rwx788 ping :smiling_imp: